datascience-notebook/Makefile

```
@@ -6,7 +6,7 @@ export PROJECT_NAME=datascience-notebook
 .PHONY: build test
 
 build:
-	docker build -t mojanalytics/${PROJECT_NAME}:${BUILD_TAG} .
+	docker build -t 593291632749.dkr.ecr.eu-west-1.amazonaws.com/${PROJECT_NAME}:${BUILD_TAG} .
```

```suggestion
	docker build -t 593291632749.dkr.ecr.eu-west-1.amazonaws.com/${PROJECT_NAME}:${IMAGE_TAG} .
```

I think this needs pulling in line with the GitHub Actions config 👍